packages/web-components/src/components/dotcom-shell/dotcom-shell-container.ts

```diff
@@ -45,127 +45,121 @@ class DDSLinkWithIcon extends LitElement {
   }
 
   /**
-   * `true` to activate the search box.
+   * `true` to activate the search box. This goes to masthead.
    */
   @property({ attribute: 'activate-search' })
   activateSearch = false;
 
   /**
-   * The profile items for authenticated state.
+   * The profile items for authenticated state. This goes to masthead.
    */
   @property({ attribute: false })
   authenticateProfileItems?: MastheadProfileItem[];
 
   /**
-   * The brand name.
+   * The brand name. This goes to masthead.
    */
   @property({ attribute: 'brand-name' })
   brandName?: string;
 
   /**
-   * The g11n collator to use for sorting contry names.
+   * The g11n collator to use for sorting contry names. This goes to footer.
    */
   @property({ attribute: false })
   collatorCountryName = new Intl.Collator();
 
   /**
-   * The throttle timeout to run query upon user input.
+   * The throttle timeout to run query upon user input. This goes to masthead.
    */
   @property({ type: Number })
   inputTimeout?: number;
 
   /**
-   * The footer links.
+   * The footer links. This goes to footer.
    */
   @property({ attribute: false })
   footerLinks: BasicLinkSet[] = [];
 
   /**
-   * The language to show in the UI.
+   * The language to show in the UI. This goes to footer.
    */
   @property({ attribute: 'lang-display' })
   langDisplay?: string;
 
   /**
-   * The language used for query.
+   * The language used for query. This goes to masthead and footer.
    */
   @property()
   language?: string;
 
   /**
-   * The legal nav links.
+   * The legal nav links. This goes to footer.
    */
   @property({ attribute: false })
   legalLinks: BasicLink[] = [];
 
   /**
-   * The locale list.
+   * The locale list. This goes to footer.
```

Comment:
Should say this is for SSR, if not allowing it to call in the client side.

Reply:
I see - Updated: https://github.com/carbon-design-system/ibm-dotcom-library/pull/3505/commits/a5f588240d67d327c90e80a21a43e3122d2814ef